src/channels/ChallengeChannels/types.ts

```diff
@@ -1,12 +1,12 @@
 import { DidPublicKey } from '@kiltprotocol/types';
 
 export interface ChallengeInput {
-  dAppEncryptionKeyId: DidPublicKey['id'];
+  dAppEncryptionKeyId: DidPublicKey['uri'];
```

Comment:
Please mark it somewhere as a followup task: we should update our naming to use URI instead of ID where appropriate